seuac-credentials-plugin/README.md

```diff
@@ -43,6 +43,7 @@ The plugin defines the following tasks:
 Task name | Depends on | Type | Description
 --- | --- | --- | ---
 `setCredentials`| - | - | Sets a credential. Invoke with `--key [Key of the credentials]` parameter.
+`clearCredentials`|-|-|Clears a credential. Invoke with `--key [Key of the credentials]` parameter. Invoke without `--key` parameter to clear all stored credentials.
```

Comment:
I added the info about the user confirmation to the readme, see https://github.com/clboettcher/seu-as-code.plugins/commit/bb6069e2eaac198a6cf13d65a5c052764c17606d